docs-ref-services/key-vault.md

```diff
@@ -14,27 +14,32 @@ ms.service: key-vault
+## Libraries for data access
 
-## Management Package
+The latest version of the Azure Key Vault library is version 4.x. Microsoft recommends using version 4.x for new applications.
 
-### Install the npm module 
+If you cannot update existing applications to version 4.x, then Microsoft recommends using version 1.x.
 
-Install the Azure Key Vault npm module
+### Version 4.x
 
-```bash
-npm install @azure/arm-keyvault
-```
+The version 4.x libraries for JavaScript are part of the Azure SDK for JavaScript. The source code for the Key Vault libraries for JavaScript is available on [GitHub](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault).
 
-### Example
+Use the following version 4.x libraries to work with certificates, keys, and secrets:
 
+| Library | Reference | Package | Source |
+|----------------------------------------|-------------------------------------------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+|    keyvault-certificates    |      [Reference](https://docs.microsoft.com/javascript/api/@azure/keyvault-certificates/?view=azure-node-latest)       |    [npm](https://www.npmjs.com/package/@azure/keyvault-certificates)    |    [GitHub](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-certificates)    |
+|    keyvault-keys    |     [Reference](https://docs.microsoft.com/javascript/api/@azure/keyvault-keys/?view=azure-node-latest)    |    [npm](https://www.npmjs.com/package/@azure/keyvault-keys)      |     [GitHub](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-keys)|
+|    keyvault-secrets   |    [Reference](https://docs.microsoft.com/javascript/api/@azure/keyvault-secrets/?view=azure-node-latest)    |    [npm](https://www.npmjs.com/package/@azure/keyvault-secrets)    |    [GitHub](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/keyvault/keyvault-secrets)    |
 
-## Samples
-- Examples for using this module in Node.js as well as browser applications can be found in the [README for the module](https://www.npmjs.com/package/@azure/arm-keyvault)
-- [Getting started with Key Vault in JavaScript](https://azure.microsoft.com/resources/samples/key-vault-node-getting-started/)
-- [Manage Azure resources and resource groups with JavaScript](https://azure.microsoft.com/resources/samples/resource-manager-node-resources-and-groups/) 
-- [Integrating Azure AD into a NodeJS web application](https://azure.microsoft.com/resources/samples/active-directory-node-webapp-openidconnect/) 
+### Version 1.x

```

Comment:
But that package below appears to be the management package (ARM), and should have a different section header. See .NET for example (IIRC, has "Management" in the title).

Reply:
See https://github.com/Azure/azure-docs-sdk-dotnet/pull/1175/files#diff-80809409a19a2a817c014a0770202dceR45

Reply:
Also, we should have a table for the v3 packages, along with links to them in the ref

Reply:
The right structure would be a 4.x.x table, a 3.x.x table and a management table

Reply:
I can't find the 3.x.x version for key vault service in this repo. @heaths , could you help me on this?